ui/app/pages/permissions-connect/permissions-connect.container.js

```diff
@@ -26,8 +26,11 @@ const mapStateToProps = state => {
     addressLastConnectedMap[key] = formatDate(addressLastConnectedMap[key], 'yyyy-M-d')
   })
 
+  const permissionsRequestId = permissionsRequest && permissionsRequest.metadata && permissionsRequest.metadata.id || null
```

Comment:
```suggestion
  const permissionsRequestId = (permissionsRequest && permissionsRequest.metadata) ? permissionsRequest.metadata.id : null
```

Reply:
This chaining of expressions is confusing—I think this is better expressed with a ternary